sos/plugins/kubernetes.py

```diff
@@ -15,40 +15,130 @@
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 from sos.plugins import Plugin, RedHatPlugin
+from os import path
 
 
 class kubernetes(Plugin, RedHatPlugin):
 
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
+    files = ("/etc/origin/master/master-config.yaml",)
+
+    option_list = [
+        ("all", "use --all-namespaces for all kubectl commands",
+            'fast', True),
+        ("describe", "capture descriptions of all kube resources",
+            'fast', True),
+        ("podlogs", "capture logs for pods", 'slow', False),
+    ]
```

Comment:
I'd suggest formatting the options list something like this:

```python
    option_list = [
        ("all", "use --all-namespaces for kubectl commands", 'fast', True),
        ("describe", "describe all kubernetes resources", 'fast', True),
        ("podlogs", "capture logs for pods", 'slow', False),
    ]
```

It requires a bit of creativity with the descriptions but options lists that fit < 80 chars are strongly preferred.